lib/util/git/index.ts

```diff
@@ -669,9 +672,13 @@ export async function commitFiles({
         }
       }
     }
-    const commitRes = await git.commit(message, [], {
-      '--no-verify': null,
-    });
+
+    const commitOptions: Options = {};
+    if (getNoVerify().includes('commit')) {
```

Comment:
@rarkins Using `getAdminConfig().gitNoVerify` here directly?

```suggestion
    if (getAdminConfig().gitNoVerify.includes('commit')) {
```

Reply:
This isn't in line with what @rarkins suggested. Going with his recommendation of removing it from the admin config.

Reply:
ok, sure, sorry.